modules/base/acl/module.py

```diff
@@ -25,494 +23,606 @@ def __init__(self, bot):
     #
```

Comment:
What does this lonely hash do here? 🤔

Reply:
It's visual separator between sections. It's not necessary, but I tend to use them from time to time, especially in bigger modules.

Reply:
I see, I try to do the same. Maybe just add label to it? Like ` # HELPER FUNCTIONS` or ` # LISTENERS`